app/services/utils/maat/other_income_base.rb

```diff
@@ -64,10 +64,16 @@ def create_other_payment(ownership_type)
 
       def income_payment_notes(ownership_type)
         other_payments_by_ownership(ownership_type).group_by { |h| h['ownership_type'] }.map do |owner, pymts|
-          "#{owner}: #{pymts.map { |p| "#{p['payment_type']}:#{p['amount']}:#{p['frequency']}" }.join(', ')}"
+          "#{owner}: #{pymts.map do |p|
+                         "#{p['payment_type']}:#{formatted_amount(p['amount'])}:#{p['frequency']}"
```

Comment:
Could we switch the second `:` to a '/' please so it reads as a 'per' 

i.e. £300.00/week